src/Manager.ts

```diff
@@ -0,0 +1,180 @@
+import { Card } from './Card';
+import { Game } from './Game';
+import { Lobby } from './Lobby';
+import { Player } from './Player';
+import { Loiterer } from './Loiterer';
+import { Operative } from './Operative';
+import { PlayerState } from './PlayerState';
+import { Broadcaster } from './Broadcaster';
+import { words } from './constants/Wordlist';
+import { GameUtility as gu } from './GameUtility';
+import { RuleEnforcer as re } from './RuleEnforcer';
+import { Team, PlayerLocation } from './constants/Constants';
+
+import ws = require('ws');
+import { shuffle } from 'lodash'
+
+export class Manager {
```

Comment:
Considering that these are new classes that folks won't be familiar with, can you add a block comment explaining what the Manager is responsible for?